hw02/PlayerController.cs

```diff
@@ -19,40 +20,82 @@ 4. Jump Speed as a float
     When the game starts, we would like to find our character. In our Unity Editor, we have a 3D model of our player and the player is 
     represented as a component in Unity. How do we grab our player controller in code and where should we write this line of code in this
     C# document (PlayerController.cs)?
+
+    We can create a new folder in Assets and name it "Scripts". 
+    Then create a new C# script and write "PlayerController.cs" document in it. 
+    Lastly drag this script to the inspector of 3D model as a component in Unity.
     */
-    
-    // Start is called before the first frame update
+
+
+    // Declaring a variable
```

Comment:
Nice job accepting user input! Here we are referring to getting the player object in the scene and getting the player controller component. We will locate the object first (Ex: gameobject.find()), then getcomponent<PlayerController>();